crates/compiler/builtins/bitcode/src/compiler_rt.zig

```diff
@@ -17,17 +17,24 @@ const v2u64 = @Vector(2, u64);
 
 // Export it as weak incase it is already linked in by something else.
 comptime {
-    @export(__muloti4, .{ .name = "__muloti4", .linkage = .Weak });
-    @export(__lshrti3, .{ .name = "__lshrti3", .linkage = .Weak });
+    // On Windows this gives lld-link: error: duplicate symbol: __muloti4
+    // >>> defined at compiler_rt.lib(compiler_rt.obj)
+    // lld-link: error: duplicate symbol: __lshrti3
+    // >>> defined at compiler_rt.lib(compiler_rt.obj)
+    if (builtin.os.tag != .windows) {
+        @export(__muloti4, .{ .name = "__muloti4", .linkage = .Weak });
+        @export(__lshrti3, .{ .name = "__lshrti3", .linkage = .Weak });
+    }
```

Comment:
`.Weak` should fix exactly this issue... Maybe there is some other linkage option that just makes this work?

Reply:
You are correct, I was able to remove this with no change.